src/libraries/pretest.proj

```diff
@@ -106,6 +106,12 @@
       <FrameworkListRootAttributes Include="FrameworkName" Value="$(SharedFrameworkName)" />
     </ItemGroup>
 
+    <ItemGroup>
+      <!-- Remove pdbs form RuntimeList.xml as they shouldn't be there -->
+      <_runtimePackLibFiles Remove="@(_runtimePackLibFiles)" Condition="'%(Extension)' == '.pdb'" />
```

Comment:
I see we are setting `IsSymbolFile` metadata on the `_runtimePackLibFiles` above. What is that used for?

Wouldn't the more appropriate fix be to set `IsSymbolFile` for the System.Private.CoreLib assembly in `_runtimePackNativeFiles`? I see symbol files are being filtered in the task here:

[https://github.com/dotnet/arcade/blob/c6a0476d79935a5a7f998c3c7ade5b30252e4d07/src/Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk/src/CreateFrameworkListFile.cs#L87-L92](https://github.com/dotnet/arcade/blob/c6a0476d79935a5a7f998c3c7ade5b30252e4d07/src/Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk/src/CreateFrameworkListFile.cs#L87-L92)

Reply:
I checked with @safern and we don't seem to be using the pdbs on wasm or xamarin runs either (which are the other places where the runtime packs are relevant) and we really only added them initially because binplace logic will copy them alongside the libraries so unless there is a strong preference, I would rather remove them in order to have our RuntimeList.xml match the one we ship. If we find that we actually could benefit in some of our test runs from having the pdbs listed there as well, I'm happy to go and change this back and add the additional metadata.

Reply:
So @eerhardt was right in that the task was going to filter the files if they had that metadata set but turns out we weren't setting it for anything as we were setting the metadata on the same place as we were doing the Item Include where the metadata isn't populated yet. I have updated this PR to actually set that metadata correctly and now I'm seeing that no PDBs are being included on the RuntimeList.xml as the task is properly doing the filtering this time. Thanks for the pointer @eerhardt